tools/geoserver/geoserver_data/psp/settings.xml

```diff
@@ -11,7 +11,7 @@
     <id>contact</id>
   </contact>
   <charset>UTF-8</charset>
-  <numDecimals>4</numDecimals>
+  <numDecimals>8</numDecimals>
```

Comment:
I believe this is the reason the draft markers were being limited to four decimal places - but this restriction only exists on our local geoserver and not the moti one.